src/modules/simulator/simulator.h

```diff
@@ -353,7 +354,8 @@ class Simulator : public ModuleParams
 	// class methods
 	int publish_sensor_topics(mavlink_hil_sensor_t *imu);
 	int publish_flow_topic(mavlink_hil_optical_flow_t *flow);
-	int publish_ev_topic(mavlink_vision_position_estimate_t *ev_mavlink);
+	template<typename T>
+	int publish_odometry_topic(T *odom_mavlink);
```

Comment:
No need for a template, just use `mavlink_message_t *odom_mavlink`

Reply:
OK got it. Thanks